multitenant/bootstrapper.go

```diff
@@ -0,0 +1,53 @@
+package multitenant
```

Comment:
I rather not have a different package for multitenancy, as it will be the default setup either if there is 1 tenant or 100+. We can leave it this way for now, but I believe it will go "potentially" to node package or similar.

Reply:
Its a limitation in bootstrapping stuff, otherwise it creates dependancy cycles, I also would have preferred to have this in the node.